app/_hub/kong-inc/confluent/overview/_index.md

```diff
@@ -0,0 +1,65 @@
+---
+nav_title: Overview
+---
+
+With Kafka at its core, [Confluent](https://confluent.io) offers complete, fully managed, cloud-native data streaming that's available everywhere your data and applications reside.
+
+<!-- I modified this a bit to point to Apacha  Kafka installations -->
+Kong also provides a Kafka Log as well as Kafka Upstream plugin for publishing logs and messages to a Apache Kafka topic.
+
+* See [Kafka Log](/hub/kong-inc/kafka-log/)
+* See [Kafka Upstream](/hub/kong-inc/kafka-upstream/)
+
+## Enable on a service-less route
+
+```bash
+curl -X POST http://localhost:8001/routes/my-route/plugins \
+    --data "name=confluent" \
+    --data "config.bootstrap_servers[1].host=localhost" \
+    --data "config.bootstrap_servers[1].port=9092" \
+    --data "config.cluster_api_key=my-api-key" \
+    --data "config.cluster_api_secret=my-api-secret" \
+    --data "config.topic=my-confluent-topic" \
+```
```

Comment:
```suggestion
```
This is a repeat of the quickstart, you don't need this.

Reply:
Removed